code/modules/food_and_drinks/food/snacks.dm

```diff
@@ -1013,10 +1013,10 @@
 	filling_color = "#C4BF76"
 	list_reagents = list("nutriment" = 2)
 
-/obj/item/weapon/reagent_containers/food/snacks/spagetti
-	name = "Spagetti"
+/obj/item/weapon/reagent_containers/food/snacks/spaghetti
+	name = "Spaghetti"
 	desc = "A bundle of raw spaghetti."
-	icon_state = "spagetti"
+	icon_state = "spaghetti"
```

Comment:
I have to know who spelled it like this now. Sure I can't spell worth a damn either without spellcheck half the time but really. Who did this?